lib/views/auth/auth_page_form.dart

```diff
@@ -45,6 +47,9 @@ class _AuthPageFormState extends State<AuthPageForm> {
       (m) => isRegisterPage ? m.registerStatus.isFetching() : m.loginStatus.isFetching(),
     );
 
+    ApiError? loginError = context.select<AuthModel, ApiError?>((m) => m.loginStatus.error);
```

Comment:
The login and register forms should be widgets in order to only dans the corresponding error instead of getting both of them